interface/usergroup/user_admin.php

```diff
@@ -605,6 +617,19 @@ class="form-control" value="<?php echo attr($frow['id']); ?>"><?php echo text($f
 
         </td>
     </tr>
+    <tr>
+        <td colspan="4">
+            <?php
+            // TODO: we eventually want to move to a responsive layout and not use tables here.  So we are going to give
+            // module writers the ability to inject divs, tables, or whatever inside the cell instead of having them
+            // generate additional rows / table columns which locks us into that format.
+            $postRenderEvent = new UserEditRenderEvent('user_admin.php', $_GET['id']);
+            error_log("rendering event");
```

Comment:
remove above debug line before bringing in

Reply:
Thanks for the catch!